common/app/model/content.scala

```diff
@@ -262,6 +265,24 @@ class Content protected (val apiContent: ApiContentWithMeta) extends Trail with
     height <- apiContent.metaData.flatMap(_.imageCutoutSrcHeight).flatMap(s => Try(s.toInt).toOption)
   } yield FaciaImageElement(src, width, height)
 
+  override lazy val slideshowImages: Boolean = {
+    apiContent.metaData.flatMap(_.json.get("slideshowImages").flatMap(_.asOpt[Boolean])).getOrElse(false)
+  }
+
+  override lazy val slideshow: Iterable[FaciaImageElement] =
+    (for {
+      x <- apiContent.metaData
+      z <- x.json.get("slideshow")
+      w = for {
+        a <- z.asInstanceOf[JsArray].value
+        y = for {
+          src <- a.asInstanceOf[JsObject].\("src").asOpt[String]
+          width <- a.asInstanceOf[JsObject].\("width").asOpt[String].flatMap(s => Try(s.toInt).toOption)
+          height <- a.asInstanceOf[JsObject].\("height").asOpt[String].flatMap(s => Try(s.toInt).toOption)
+        } yield FaciaImageElement(src, width, height)
+      } yield y
+    } yield w).getOrElse(Nil).flatten
+
```

Comment:
Can we use more meaningful names? x, y, z etc save space but it's not immediately clear what we're yielding.